file-avail-s3.md

```diff
@@ -0,0 +1,235 @@
+
+
+## A. Intended Use
+
+Our S3 data storage is only for input and output files that
+are individually too large to be managed by our other data movement
+methods, HTCondor file transfer or SQUID, and when these files are
+expected to be accessed outside of CHTC. This includes individual input files
+greater than 100MB and individual output files greater than 3-4GB.
+
+Users are expected to abide by this intended use expectation and follow the
+instructions for using S3 written in this guide (e.g. files placed
+in S3 should ALWAYS be listed in the submit file).
+
+## B. Access to S3 Data Storage
+
+Any one with a CHTC account whose data meets the intended use above
+can request access to CHTC's S3 data storage. A Research
+Computing Facilitator will review the request and follow up. If
+appropriate, S3 bucket creation will be enabled and a quota set on
+your account. Quotas are based on individual user needs; if a larger quota
+is needed, email chtc@cs.wisc.edu with your request.
+
+## C. User Data Management Responsibilities
+
+As with all CHTC file spaces:
+
+- **Keep copies**: Our S3 buckets are not backed up and have the
+possibility of data loss; keep copies of ANY and ALL data in S3
+buckets in another, non-CHTC location.
+- **Remove data**: We expect that users remove data from S3 buckets AS
+SOON AS IT IS NO LONGER NEEDED FOR ACTIVELY-RUNNING JOBS.
+- **Monitor usage and quota**: Your account has both a size and
+number of files quota that applies across all buckets owned by your
+account. Quota changes can be requested by emailing chtc@cs.wisc.edu.
+
+CHTC staff reserve the right to remove S3 buckets or revoke bucket
+creation permission at any time.
+
+## D. Data Access Within Jobs
+
+Data in a CHTC S3 bucket is available anywhere (including OSG) as long
+as the version of HTCondor running on the execute machines support
+jobs that use S3 URLs.
+
+Data in CHTC S3 buckets are owned by the user (or a set of users), and
+only the user's (or users') own jobs can access these files.
+
+# 2. Staging Large Data
+
+In order to stage data in an S3 bucket for use on CHTC's HTC system:
+
+- **Get S3 bucket creation access**: Bucket creation access is granted by request.
+- **Create an S3 bucket**: Create a bucket that will contain the data for your project.
+- **Reduce file counts**: Combine and compress files that are used together.
+- **Use the transfer server**: Upload your data to your bucket via our dedicated file transfer server.
+- **Remove files after jobs complete**: Data in S3 buckets are quota controlled and not backed up.
+
+## A. Get S3 Bucket Creation Access
+
+CHTC S3 bucket creation access is granted by request. If you think you need
+to create S3 buckets, email CHTC's Research Computing Facilitators (chtc@cs.wisc.edu).
+
+## B. Create an S3 Bucket
+
+Buckets can be created on a CHTC submit server or the CHTC transfer server
+using the `mc` command:
+
+``` {: .term}
+$ mc mb chtc/my-bucket-name
```

Comment:
Maybe change the prompt here to something like: `[alice@transfer]$`, to be clear where these commands should happen?

Reply:
> Jason, I'd be interested in your comments of the general outline -- I reworked the Gluster guide when we switched to `/staging` and I'd be interested to know if this general structure worked well for you here or if you'd rearrange if you didn't have that guide to go off of.

Hard to say now that I've done it what changes I would make, it seemed pretty straight forward to me. I do like how it starts high level and then goes in to the details only after listing the recommended usage and use policy. I'll let you know if I have any more feedback after staring at it more.

Reply:
> Maybe change the prompt here to something like: `[alice@transfer]$`, to be clear where these commands should happen?

Done for `mc` examples. I think most other examples are somewhat universal (at least in the *nix realm).